public/css/list/item-table.less

```diff
@@ -62,3 +62,27 @@
     width: 1em;
   }
 }
+
+.controls .user-list,
+.controls .usergroup-list {
+  .usergroup-ball,
+  .user-ball {
+    height: 1.5em;
+    width: 1.5em;
+    line-height: ~"calc(1.5em - 4px)";
+    display: block;
+  }
+
+  .title br {
+    display: none;
+  }
+
+  .list-item > .col {
+    padding: .25em 0;
+  }
+}
```

Comment:
Doesn't belong here. This has nothing to do with ordinary item tables. Put that into a separate file for user- and usergroup-lists.

Reply:
✅